tensorflow/python/training/saver_test.py

```diff
@@ -696,7 +684,10 @@ def _testPartitionedVariables(self, use_resource):
     var_full_shape = [10, 3]
     # Allows save/restore mechanism to work w/ different slicings.
     var_name = "my_var"
-    saved_path = os.path.join(_TestDir("partitioned_variables"), "ckpt")
+    saved_dir = os.path.join(self.get_temp_dir(), "partitioned_variables")
```

Comment:
These two lines are very repetitive in this test class. Can't we just create a temporary directory for each 

def _test_dir(self, test_name):
  test_dir = os.path.join(self.get_temp_dir(), test_name)
  gfile.MakeDirs(test_dir)
  return test_dir